### PR TITLE
chore[DST-60]: change depricated API

### PR DESCRIPTION
**Pull request by sarahgm**

### CODE REVIEW

File: packages/components/src/Table/Table.tsx [hunk 1]
```diff
@@ -90,17 +90,21 @@ export const Table: Table = ({
...
-              {[...row.childNodes].map(cell =>
-                cell.props?.isSelectionCell ? (
-                  <TableCheckboxCell key={cell.key} cell={cell} />
-                ) : (
-                  <TableCell key={cell.key} cell={cell} />
-                )
-              )}
-            </TableRow>
-          ))}
+          {/* {...bodyElements.map(row as  => ( */}
```
Comment (sebald) [inline packages/components/src/Table/Table.tsx:93]: Remove? 😀

File: packages/components/src/Table/TableRow.tsx [hunk 2]
```diff
@@ -22,13 +22,10 @@ export const TableRow = ({ children, row }: TableRowProps) => {
   const ref = useRef(null);
   const { interactive, state, ...ctx } = useTableContext();
 
-  const { variant, size } = row.props;
```
Comment (sebald) [inline packages/components/src/Table/TableRow.tsx:25]: Is this no longer possible?

**Review by sebald — CHANGES_REQUESTED** [review]

File: packages/components/src/Table/Table.tsx (hunk 1 above)
Comment (sarahgm) [reply packages/components/src/Table/Table.tsx]: ups

File: packages/components/src/Table/TableRow.tsx (hunk 2 above)
Comment (sarahgm) [reply packages/components/src/Table/TableRow.tsx:25]: to be honest, I don't actually see the use case

Comment (sarahgm) [reply packages/components/src/Table/TableRow.tsx:25]: should I add this question comment also again? ;)

Comment (sebald) [reply packages/components/src/Table/TableRow.tsx:25]: the idea was to have a highlighted row, maybe. but we can also readd it later if necessary :)

**Review by sebald — APPROVED** [review]

**Pull request merged**